extensions/elytron-security-ldap/deployment/src/test/java/io/quarkus/elytron/security/ldap/LdapSecurityRealmTest.java

```diff
@@ -128,15 +128,19 @@ public void testJaxrsUserRoleSuccess() {
         setupAuth("standardUser", "standardUserPassword")
                 .when().get("/jaxrs-secured/subject/secured").then()
                 .statusCode(200)
-                .body(equalTo("standardUser"));
+                .body(equalTo(expectedStandardUserName()));
```

Comment:
I had to do it because there are 4 concrete test suite classes extending this abstract class